Separate enums into singleton

ojas-sanghi opened the pull request:
Also move out World scenes since they're not singletons (yet)

ojas-sanghi committed:
Separate enums into singleton

Also move out World scenes since they're not singletons (yet)

cpostbitbuckets commented on src/GameObjects/Player.gd:
```diff
@@ -10,18 +10,11 @@ var resources = {
-	"missile": missile_enum.missile1,
-	"laser": laser_enum.laser1,
-	"shield": shield_enum.shield1
+	"mine": Enums.raw_enum.mine1,
```
since we are referring to these as `Enums.raw_enum`, can we just call it `Enums.raw`?

ojas-sanghi committed:
Change enums name

Remove the "_enum" from them since they're already in the "Enums.gd" file and are referred to as "Enums.science", for example
